PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/LoadJsonConfigFile.java

```diff
@@ -23,22 +30,31 @@ public class LoadJsonConfigFile {
     private static LoadJsonConfigFile instance;
     private JSONObject jsonObject;
 
-    final String CONFIG_FILE_PATH = "META-INF/ruleConfigs.json";
+    static final String CONFIG_FILE_PATH = "META-INF/ruleConfigs.json";
```

Comment:
Do we need this to be private?

Reply:
its accessed by Logging in other rules so its package-private

Reply:
I believe this can be constant as this does not change with every rule file.

Reply:
do you mean     static final String CONFIG_FILE_PATH = "META-INF/ruleConfigs.json";? its already a constant? unless I'm missing something

Reply:
or do you mean jsonObject? I'll make that a constant since it fits what you said